index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Muscle-Blaze</title>
+</head>
+<body>
+   
+    <header>
+        <h1 class="logo"><a href="">Muscle Blaze</a></h1>
```

Comment:
Please use some meaningful class names !!? How a heading can be a `logo` ??

Reply:
I just added this by mistake